server/tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "stripInternal": true,
     "paths": {
       "firebase": ["./node_modules/firebase"],
-      "rxjs/*": ["./node_modules/rxjs/*"]
+      "rxjs/*": ["./node_modules/rxjs/*"],
+      "models/*": ["./src/models/*"]
```

Comment:
That's quite nice! 👍

Might be worth exploring where else we can use this feature in the project.

Reply:
this is analogous to the old `alias` feature in webpack. I like this because you can get rid of `../../../../` when you have a deeply nested folder structure and just reference a folder like you would a node_module